app/src/main/java/de/perflyst/untis/activity/ActivityMain.java

```diff
@@ -26,51 +26,28 @@
 import android.text.Editable;
 import android.text.TextUtils;
 import android.text.TextWatcher;
-import android.view.Menu;
-import android.view.MenuItem;
-import android.view.MotionEvent;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.GridView;
-import android.widget.ImageView;
-import android.widget.LinearLayout;
-import android.widget.RelativeLayout;
-import android.widget.TextView;
-
+import android.view.*;
+import android.widget.*;
 import de.perflyst.untis.BuildConfig;
 import de.perflyst.untis.R;
 import de.perflyst.untis.adapter.AdapterGridView;
 import de.perflyst.untis.adapter.AdapterTimetable;
 import de.perflyst.untis.adapter.AdapterTimetableHeader;
 import de.perflyst.untis.fragment.FragmentDatePicker;
-import de.perflyst.untis.utils.Conversions;
-import de.perflyst.untis.utils.DateOperations;
-import de.perflyst.untis.utils.ElementName;
-import de.perflyst.untis.utils.ListManager;
-import de.perflyst.untis.utils.SessionInfo;
+import de.perflyst.untis.notification.StartupReceiver;
+import de.perflyst.untis.utils.*;
 import de.perflyst.untis.utils.timetable.TimegridUnitManager;
-
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLDecoder;
-import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.Collections;
-import java.util.List;
-import java.util.Locale;
-import java.util.NoSuchElementException;
-
-import de.perflyst.untis.utils.ThemeUtils;
+import java.util.*;
```

Comment:
Same here as above, should we keep it?

Reply:
What's your point about not using *? In my opinion the import block shrinks which reduces the file size.
[Here](https://coderanch.com/wiki/659753/Wild-Card-Specific-Imports) are some arguments for and against wildcard imports

Reply:
Well, ok. As long as there is no other library who may conflict with these we can keep it.